app/controllers/api/public/v1/school_suggestions_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module API
+  module Public
+    module V1
+      class SchoolSuggestionsController < API::Public::V1::ApplicationController
```

Comment:
Hate to be a pain but can we move this to just `API::SchoolSuggestionsController`. It won't be publicly available or documented but internal use

Reply:
Coolio